### PR TITLE
Blog - Docker Slim

### PR DESCRIPTION
**Pull request by msali123**
Done with all checks

### CODE REVIEW

File: blog/_posts/2022-07-25-docker-slim.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+
+## Installing Docker Slim
+
+Docker Slim currently works with Linux and Mac operating systems. It can be installed by downloading the [binary packages](https://github.com/docker-slim/docker-slim#downloads) or utilizing a package manager, like [Homebrew](https://github.com/docker-slim/docker-slim#homebrew). The tool is also available to be pulled as a [Docker image](https://hub.docker.com/r/dslim/docker-slim/tags), and Docker Slim offers a [software-as-a-service (SaaS)](https://portal.slim.dev/login?invitecode=invite.1s85zlfnYX0p5TT1XKja49pAHbL) platform to utilize its functionalities.
+
+For the purpose of this article, an Ubuntu (18.04 LTS) environment was used with Docker Slim installed using the prepared Bash script available on the official [Docker Slim GitHub repo](https://github.com/docker-slim/docker-slim/blob/master/scripts/install-dockerslim.sh) and the following CLI command:
+
+~~~{.bash caption=">_"}
```
Comment (adamgordonbell) [inline blog/_posts/2022-07-25-docker-slim.md:50]: See how the caption is overlapping with the command and also how there is a scroll bar. 

Can we format this better somehow?

Screen Shot 2022-07-06 at 3 26 01 PM

My suggestion would be to use line breaks and `\` to make it go across multiple lines:
```
curl -sL \
   https://raw.githubusercontent.com/docker-slim/docker-slim/master/scripts/install-dockerslim.sh \
   | sudo -E bash -
```

File: blog/_posts/2022-07-25-docker-slim.md [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+### Security
+
+Docker, and in general, containerized applications, can often be more secure than traditional local applications. However, there are considerations to note, such as the permissions allowed by your kernel, the interaction between Docker, your containers and the file system, and any unnecessary loopholes in your configuration profile.
+These concerns can be alleviated by adding another safety layer with a unique security configuration profile to your container.
+
+Docker Slim automatically generates [AppArmor](https://apparmor.net/) and [Seccomp](https://docs.docker.com/engine/security/seccomp/) security profiles when the `build` or `profile` commands are used. These security profiles will be specific to your images and their functionality.
+
+You can use the security profile generated in the previous build command using the following syntax:
+
+~~~{.bash caption=">_"}
```
Comment (adamgordonbell) [inline blog/_posts/2022-07-25-docker-slim.md:157]: Similar here. Maybe something like this:
```
docker run -it --rm -d -p 8080:80 \
--security-opt apparmor:nginx-apparmor-profile nginx.slim
```

**Review by adamgordonbell — CHANGES_REQUESTED** [review]

File: blog/_posts/2022-07-25-docker-slim.md (hunk 2 above)
Comment (msali123) [reply blog/_posts/2022-07-25-docker-slim.md:159]: Resolved
Sorry for the inconvenience in future I'll break all the code block that have a scroll bar

File: blog/_posts/2022-07-25-docker-slim.md (hunk 1 above)
Comment (msali123) [reply blog/_posts/2022-07-25-docker-slim.md:50]: Resolved

File: blog/_posts/2022-07-25-docker-slim.md (hunk 2 above)
Comment (adamgordonbell) [reply blog/_posts/2022-07-25-docker-slim.md:159]: That's great, Thank you. Not an inconvenience at all.

**Review by adamgordonbell — APPROVED** [review]

**Comment by msali123** [comment]
@jalletto done with review

**Comment by jalletto** [comment]
Commit ffe7e884acd6e7d198c11729e3427668f8c87ea7 deployed to https://62c66d2acfa0bf3da9624ced--ephemeral-narwhal-3ac3b2.netlify.app/

**Pull request merged**